README.md

```diff
@@ -1,34 +1,142 @@
-# W3C Http Signatures Working Group Test Suite
+W3C Signing HTTP Messages Working Group Test Suite
+==================================================
 
 This repository contains the W3C
-[Http Signatures](https://tools.ietf.org/html/draft-cavage-http-signatures-11) test suite.
+[Signing HTTP Messages draft v11](https://tools.ietf.org/html/draft-cavage-http-signatures-11) test suite.
 Any conforming implementation MUST pass all tests in the test suite.
 
-There are multiple test suites, each of which is detailed below.
+# Signing HTTP Messages Tests
 
-## Http Signatures Version 11 Tests
+The test suite will check an implentation that generates and validates signatures
+compliant with
+[Signing HTTP Messages](https://tools.ietf.org/html/draft-cavage-http-signatures)
+to ensure conformance with the specification.
 
-This test suite will check any application that generates [Http Signaturesl](hhttps://tools.ietf.org/html/draft-cavage-http-signatures-11) Headers to
-ensure conformance with the specification.
+This suite requires a functioning installation of
+[nodejs](https://nodejs.org) (>v8.12), and specifically
+the ``npm`` commmand (>v6.4).
```

Comment:
```suggestion
the `npm` commmand (>v6.4).
```

Reply:
The double backticks are used all over.  Why not just single?

Reply:
A habit of mine for all quotes. Allows free usage of single backtick even within a ``code` block``, which would otherwise signal the end of the `code` block`.